gubernator/templates/filtered_log.html

```diff
@@ -13,11 +13,14 @@
 <input type="hidden" name="logfile" value="{{log_file}}">
 <input type="hidden" name="pod" value="{{pod}}">
 <input type="hidden" name="junit" value="{{junit}}">
+<hr>
+<input type="checkbox" name="wrap"{% if wrap %} checked{% endif %}> Line wrapping on
+<hr>
 <label><input type="checkbox" name="UID"{% if uid %} checked{% endif %}> Highlight Pod UID</label><br>
 <label><input type="checkbox" name="Namespace"{% if namespace %} checked{% endif %}> Highlight Namespace</label>
 </form>
 % if log
-<pre>
+{% if wrap %}<pre>{% else %}<pre class="nowrap">{% endif %}
```

Comment:
[Some kubelet logs](https://test-dot-k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/kubelet-gce-e2e-ci/6828/nodelog?logfile=kubelet.log&pod=test-webserver-2fbf51bc-4e3a-11e6-b8f1-42010af00022&junit=junit_02.xml) have a lot of redundancy, others don't.

You can use a snippet like `<button onclick="document.getElementById('log').classList.toggle('nowrap')|1">Toggle Line Wrap</button>` to make this work without reloading the page.